### ISSUE TITLE
Build failed on Mac

### ISSUE THREAD

**Issue by oterral** [camptocamp/ngeo#2627]
Running a `make dist` on mac from a clean clone failed.

```
mkdir -p .build/externs/
wget -O .build/externs/angular-1.6-q_templated.js https://raw.githubusercontent.com/google/closure-compiler/`grep ^closure-compiler= github_versions | cut --delimiter = --fields 2`/contrib/externs/angular-1.6-q_templated.js
cut: illegal option -- -
usage: cut -b list [-n] [file ...]
       cut -c list [file ...]
       cut -f list [-s] [-d delim] [file ...]
--2017-05-15 11:37:01--  https://raw.githubusercontent.com/google/closure-compiler//contrib/externs/angular-1.6-q_templated.js
Resolving raw.githubusercontent.com... 151.101.36.133
Connecting to raw.githubusercontent.com|151.101.36.133|:443... connected.
HTTP request sent, awaiting response... 404 Not Found
2017-05-15 11:37:02 ERROR 404: Not Found.

make: *** [.build/externs/angular-1.6-q_templated.js] Error 8
```
The closure version is not taken from the github-versions file.

OSX 10.12.4

**Comment by fredj** [comment]
Can you try to replace `cut --delimiter = --fields 2` with `cut -d = -f 2` ?

**Comment by fredj** [comment]
https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/cut.1.html

**Comment by oterral** [comment]
Thx it fixes theissue but antoher is showing up:
```
npm install
npm ERR! git rev-list -n1 during_filter: fatal: ambiguous argument 'during_filter': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 during_filter: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 during_filter: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 during_filter: 
npm ERR! git rev-list -n1 during_filter: fatal: ambiguous argument 'during_filter': unknown revision or path not in the working tree.
npm ERR! git rev-list -n1 during_filter: Use '--' to separate paths from revisions, like this:
npm ERR! git rev-list -n1 during_filter: 'git <command> [<revision>...] -- [<file>...]'
npm ERR! git rev-list -n1 during_filter: 
npm ERR! git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab: Cloning into bare repository '/Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab'...
npm ERR! git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab: Warning: Permanently added the RSA host key for IP address '192.30.253.112' to the list of known hosts.
npm ERR! git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab: Permission denied (publickey).
npm ERR! git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab: fatal: Could not read from remote repository.
npm ERR! git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab: 
npm ERR! git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab: Please make sure you have the correct access rights
npm ERR! git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab: and the repository exists.
npm ERR! Darwin 16.5.0
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "install"
npm ERR! node v6.10.0
npm ERR! npm  v3.10.10
npm ERR! code 128

npm ERR! Command failed: git clone --template=/Users/kogisiwi/.npm/_git-remotes/_templates --mirror git@github.com:fredj/openlayers.git /Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab
npm ERR! Cloning into bare repository '/Users/kogisiwi/.npm/_git-remotes/git-github-com-fredj-openlayers-git-during-filter-99e320ab'...
npm ERR! Warning: Permanently added the RSA host key for IP address '192.30.253.112' to the list of known hosts.
npm ERR! Permission denied (publickey).
npm ERR! fatal: Could not read from remote repository.
npm ERR! 
npm ERR! Please make sure you have the correct access rights
npm ERR! and the repository exists.

```

**Comment by oterral** [comment]
The ol version point to a branch that doesn't exist anymore.

**Comment by fredj** [comment]
will be fixed with #2624

**Comment by oterral** [comment]
perfect thx. It works now.
